internal/forge/github/comment.go

```diff
@@ -205,11 +205,16 @@ func (f *Repository) ListChangeComments(
 			}
 
 			for _, node := range q.Node.PullRequest.Comments.Nodes {
+				isNavigationComment := true
```

Comment:
one nit: this isn't necessarily a navigation comment. At this layer of abstraction, it's just a "match".

```suggestion
				match := true
```